cypress/Dockerfile

```diff
@@ -1,17 +1,20 @@
 ARG NODE_VERSION='20.9.0'
-ARG CYPRESS_VERSION='13.3.1'
+ARG CYPRESS_VERSION='13.5.0'
```

Comment:
Looks like this env variable is no longer used.  Could this be integrated into line 15 to drive the version of the Cypress image?